jetson/odrive_bridge/src/__main__.py

```diff
@@ -304,8 +304,10 @@ def publish_encoder_helper(axis):
     msg = DriveVelData()
 
     usb_lock.acquire()
-    msg.current_amps = modrive.get_iq_measured(axis)
-    msg.vel_m_s = modrive.get_vel_estimate(axis)
+
+    encoder_multiplier = -1
+    msg.current_amps = modrive.get_iq_measured(axis) * encoder_multiplier

```

Comment:
a bit strange to be multiplying 'encoder multiplier' by the current, maybe change to direction multiplier or smth?

Reply:
yea sure makes sense